frontend/src/component/menu/routes.ts

```diff
@@ -66,8 +66,8 @@ export const routes: IRoute[] = [
         title: 'Insights',
         component: ExecutiveDashboard,
         type: 'protected',
-        menu: { mobile: true, advanced: true },
```

Comment:
Why are we removing `advanced: true` here? What does that mean?

Reply:
Advanced is the dropdown menu `Configuration`. So it was on the nav bar but also the dropdown

Reply:
Right, so it was a duplicate, and it's not supposed to be?